stats/opentelemetry/e2e_test.go

```diff
@@ -127,12 +95,11 @@ func (s) TestMethodTargetAttributeFilter(t *testing.T) {
 		// Will allow duplex/any other type of RPC.
```

Comment:
This test is only testing the method attribute filter now. The test name and comment needs updating.

Reply:
Ah, good point. Switched.

Reply:
s/The method and target filter set should bucket/The method filter set should bucket/

Reply:
Argh whoops haha. Switched.